backend/exchanges/exchanges.go

```diff
@@ -21,6 +21,22 @@ import (
 	"github.com/digitalbitbox/bitbox-wallet-app/util/logging"
 )
 
+// ErrorCode are errors that are represented by an error code. This helps the frontend to translate
+// error messages.
+type ErrorCode string
+
+func (e ErrorCode) Error() string {
+	return string(e)
+}
+
+const (
+	// ErrAddressNotFound is returned if an account is being added which already exists.
```

Comment:
the comment does not seem to match the name of the error. looking at its use, it seems the comment is wrong?

Reply:
Sorry, mispasted line. Fixed now